src/bin/vip-wp.js

```diff
@@ -225,6 +226,12 @@ commandWrapper( {
 
 					commandStreams.stdoutStream.unpipe( process.stdout );
 
+					// Need a newline - WP CLI doesn't always send one :(
+					// https://github.com/wp-cli/wp-cli/blob/779bdd16025cb718260b35fd2b69ae47ca80cb91/php/WP_CLI/Formatter.php#L129-L141
+					if ( line.includes( '--format=count' ) || line.includes( '--format=ids' ) ) {
```

Comment:
Do we care that this won't work if `--format="count"`?

Reply:
Alternatively, can we fix in the API instead?

Reply:
I guess we should handle quotes there too for consistency.

My worry is introducing "non-standard" behavior when piping output to other processes so probably better to leave it "raw" and just fix the interactive experience. Though this does affect one-off commands too (but not as badly).

It _might_ actually need to be fixed at the WP-CLI level ultimately. Need to do some more research.

Reply:
Hmm:

```
wp post list --posts_per_page=10 --format="count"
Error: Parameter errors:
 Invalid value specified for 'format' (Render output in a particular format.)
```

Reply:
Handled quoted versions in 8feb1ef

Reply:
Probably also need `--format='count'` :)

Reply:
Heh, added in b93505f